### ISSUE TITLE
MDL paradox

### ISSUE THREAD

**Issue by tarcadius** [IM3S/LDAR_Sim#37]
OGI is used to generate input distributions, then is used to detect leaks.

So, either all leaks should be detectable by OGI (probability of detection = 1 at all times)

OR

We try and reconstruct the original distribution that would have existed using a lognormal function in some way in order to replicate what the original distribution would have looked like if the empirical sample was collected by a perfect instrument.

@tbarchyn  thoughts?
@KeeganShaw-GIS easy way to do this?

**Comment by tbarchyn** [comment]
I am in favour of making the model real, not just what can be detected by OGI. This allows large scale measurements function properly and it can be used as a tool for emissions (not just LDAR), for tasks like integrating different scales of measurements. 

This explicitly disambiguates measurements of leaks and the measurement tech used from the actual real leak population. It will jack the number of small leaks up huge though and we will need to simulate the part of the curve below minimum measured OGI measurement.

**Comment by tarcadius** [comment]
I agree. This might be a job for a math person... remembering some of Arvind's older papers that include M21 distributions, which could help...

**Comment by tbarchyn** [comment]
Here are some notes on this while it is in my head:
1) Leaks measured by whatever tech in whatever campaign.

2) Leaks 'corrected' for MDL curve - this will jack up the number of small leaks where the curve drifts down from 1.0. The logic is something with a MDL of 0.5 means that half of the leaks at that rate are being seen and if the tech found 5 leaks, there was actually 10 leaks because 5 were found, and 5 were not.

3) Somewhere deep on the MDL curve (0.25 maybe), it is assumed that the system cannot reliably measure leaks and the information ends. In other words, leaks below where the detection probability of 0.25 are not visible with the tech. This line has to be drawn somewhere because the uncertainty probably gets too large.

4) Then, use the corrected leak counts as the input for LDAR sim (this should be a 'preprocessing' step I think). This way LDAR sim operates with real counts that are not all wrapped up in the biases of the measurement equipment.

This does not solve the issue of leaks that are below the MDL of the measurement technology - different technologies are required to get this lined up. The easiest one is facility or equipment scale tech - and really it should be done on the same sites in the same campaign, but this is probably not possible without a purpose built measurement campaign.

1) Use the real or predicted leak counts relative to facility size to create a 'known' visible leak emissions rate. This should be less than the facility emissions. Ideally add in emissions rates from known vents.

2) Then the difference is the sum of all the small leaks that are unmeasurable with OGI (but exist!). Because these are invisible and not really countable - they can either (a) be lumped into little leaks below the MDL of OGI, or (b) some equipment relative rate of 'ghost emissions' that are not possible to attribute - but exist!

This is a real issue because I think the Sniffers from Europe are all about using extreme sensitivity equipment and care to kill all those little leaks that OGI misses -  and allowing people to explore the emissions and cost ramifications of that general approach (even if people are addicted to super-emitters mostly these days) seems quite important.
